conf/messages.en

```diff
@@ -113,10 +113,14 @@ dateGoodsUnloaded.heading = When were the goods unloaded?
 dateGoodsUnloaded.hint = For example, 15 08 2022.
 dateGoodsUnloaded.error.required.all = Enter the date the goods were unloaded
 dateGoodsUnloaded.error.required.multiple = The date must include a {0} and {1}
-dateGoodsUnloaded.error.required = The date must include a {0}
-dateGoodsUnloaded.error.invalid.all = The date must only include numbers 0 to 9
-dateGoodsUnloaded.error.invalid.multiple = The date {0} and {1} must only include numbers 0 to 9
-dateGoodsUnloaded.error.invalid = The date {0} must only include numbers 0 to 9
+dateGoodsUnloaded.error.required.day = The date must include a day
+dateGoodsUnloaded.error.required.month = The date must include a month
+dateGoodsUnloaded.error.required.year = The date must include a year
+dateGoodsUnloaded.error.invalid.all = The date must be a real date
+dateGoodsUnloaded.error.invalid.multiple = The date must be a real date
+dateGoodsUnloaded.error.invalid.day = The day must only include numbers 1 to {0}
+dateGoodsUnloaded.error.invalid.month = The month must only include numbers 1 to 12
+dateGoodsUnloaded.error.invalid.year = The year must be a real year
```

Comment:
Shouldn't this be dateGoodsUnloaded.error.max.date = The date must be today or in the past
As per the content doc

Reply:
@vajarova looks like that is there? Line 125. Hasn't changed.

Reply:
Ignore